doc/source/io.rst

```diff
@@ -5208,82 +5208,105 @@ easy conversion to and from pandas.
 Performance Considerations
 --------------------------
 
-This is an informal comparison of various IO methods, using pandas 0.13.1.
+This is an informal comparison of various IO methods, using pandas 0.20.3.
```

Comment:
Can you maybe put a stronger warning here that the timings are machine dependent and you should not look at small differences?

Reply:
Done